perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1369,12 +1382,14 @@ sub redirect_or_confirm_creation : Private {
 
     # If confirmed send the user straight there.
     if ( $report->confirmed ) {
-        # Subscribe problem reporter to email updates
-        $c->forward( 'create_reporter_alert' );
-        if ($c->stash->{contributing_as_another_user}) {
-            $c->send_email( 'other-reported.txt', {
-                to => [ [ $report->user->email, $report->name ] ],
-            } );
+        if ($report->user->email) {
+            # Subscribe problem reporter to email updates
+            $c->forward( 'create_reporter_alert' );
```

Comment:
This signed you up for an alert even if phone-only, so that if email later got added to the account, you'd then start getting the email alerts. Dunno if we want to keep that?